src/psyclone/configuration.py

```diff
@@ -314,6 +319,14 @@ def load(self, config_file=None):
         else:
             self._psyir_root_name = self._config['DEFAULT']['PSYIR_ROOT_NAME']
 
+        # Read the valid PSyData class prefixes:
+        try:
+            # Convert it to a list of strings:
+            self._valid_psy_data_prefix = \
+                self._config["DEFAULT"]["VALID_PSY_DATA_PREFIXES"].split()
```

Comment:
Done indeed. I couldn't use the one from pattern_tools, since as far as I can see it accepts a '$', which we don't want.

I had some problem with testing: when I specified **`cd"`** as a prefix (i.e. one double quote in the name), the error message used "...'{0}'..." with the prefix ... then suddenly the single quotes were escaped.  I couldn't really figure out where this was coming from, so I just tested for both strings in the assert: one with just the single quotes, and one with escaped single quotes. Suggestions welcome, I gave up 😁